test/cluster/test_tablets_merge.py

```
@@ -652,3 +652,111 @@ async def finished_merge():
+        await manager.enable_tablet_balancing()
+
+        # Wait for merge to be finalized (tablet count changes)
+        started = time.time()
+        while True:
+            tablet_count = await get_tablet_count(manager, server, ks, 'test')
+            if tablet_count == target_tablets:
+                break
+            assert time.time() - started < 120, f'Timeout waiting for merge (current={tablet_count}, target={target_tablets})'
+            await asyncio.sleep(0.5)
+
+        logger.info(f"Merge done, tablet count = {tablet_count}")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)